docs/included.md

```diff
@@ -13,6 +13,18 @@
   - [API](api.md#scores.continuous.additive_bias)
   - [Tutorial](project:./tutorials/Additive_and_multiplicative_bias.md)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Consistent Expectile Score
+  - [API](api.md#scores.continuous.consistent_expectile_score)
+  - [Tutorial](project:./tutorials/Consistent_Scores.md)
+  - [Gneiting (2011)](https://doi.org/10.1198/jasa.2011.r101388); [Taggart (2022)](https://doi.org/10.1002/qj.4206)
```

Comment:
```suggestion
  - [Gneiting (2011)](https://doi.org/10.1198/jasa.2011.r10138); [Taggart (2022)](https://doi.org/10.1002/qj.4206)
```
Looks like there is an extra 8 at the end of the DOI (doesn't currently resolve in included.md as rendereing on 228 branch). (Assuming it is the Making and Evaluating Point Forecasts article)